Declare all the work_str vars static since they'll conflict badly
with each other if we don't.  pcalc.y doesn't actually use work_str.

Patch by Mike Frysinger <vapier@gentoo.org>

--- pcalc-000/funct.c
+++ pcalc-000/funct.c
@@ -50,7 +50,7 @@
 
 /* -------- Strings: ----------------------------------------------------- */
 
-char    work_str[2000];
+static char    work_str[2000];
 
 /* -------- Implementation: ---------------------------------------------- */
 
--- pcalc-000/pcalc.y
+++ pcalc-000/pcalc.y
@@ -37,7 +37,7 @@
 
   extern  FILE * yyin ;
 
-  char    work_str[128];
+  //static char    work_str[128];
 
   %}
 
--- pcalc-000/print.c
+++ pcalc-000/print.c
@@ -31,7 +31,7 @@
 
 /* -------- Implementation: ---------------------------------------------- */
 
-char    work_str[128];
+static char    work_str[128];
 
 void    print_num(double var)
 
--- pcalc-000/store.c
+++ pcalc-000/store.c
@@ -27,7 +27,7 @@
 
 /* -------- Implementation: ---------------------------------------------- */
 
-char    work_str[128];
+static char    work_str[128];
 
 int     store(char *file, char *name, double var)
 
